wordpress-mobile/WordPress-iOS
# [Site creation] Style WebAddressWizardContent

Fixes #10480 

Before and after:
![web-cell-before-after](https://user-images.githubusercontent.com/2722505/51153328-aef04680-18aa-11e9-9300-f517ada454b6.jpg)

iPhone X:
![simulator screen shot - iphone xr - 2019-01-15 at 09 19 37](https://user-images.githubusercontent.com/2722505/51153341-bca5cc00-18aa-11e9-87a8-df63178319c3.png)

iPhone SE:
![simulator screen shot - iphone se - 2019-01-15 at 09 29 03](https://user-images.githubusercontent.com/2722505/51153346-c3344380-18aa-11e9-921e-4f6ff4f77b5d.png)

iPad Pro 11 (2018):
![simulator screen shot - ipad pro 11-inch - 2019-01-15 at 09 43 54](https://user-images.githubusercontent.com/2722505/51153361-d2b38c80-18aa-11e9-9e28-41308d79bd54.png)


This PR changes:
- [ ] Update styles of cells
- [ ] Scroll header up when the textfield is in focus
- [ ] Make header and table view honour the layout guides

To test:
- Check the branch
- Navigate to Enhanced Site Creation until the Web address selection screen is reached.

Update release notes:
- Nothing worth mentioning in the release notes.

There is significant duplication between WebAddressWizardContent and VerticalsWizardContent. I have added a ticket to look into it later on ( #10821 ), and see if we can find a way to remove some of that duplication
